test/sslapitest.c

```diff
@@ -225,133 +190,103 @@ static int test_keylog_output(char *buffer, const SSL *ssl,
                 server_handshake_secret_count++;
             else if (strcmp(token, "CLIENT_TRAFFIC_SECRET_0") == 0)
                 client_application_secret_count++;
-            else if (strcmp(token, "SERVER_TRAFFIC_SECRET_0") == 0)
+            else
```

Comment:
No, leave the `if ...` part, if it's absurd now.  There's gonna be other tokens, and having to remember to add another than the one you're about to add makes the job harder.